pyext/src/exhaust.py

```diff
@@ -269,14 +269,15 @@ def main():
 
         # Do clustering at each cutoff
         pvals, cvs, percents = clustering_rmsd.get_clusters(
-                cutoffs_list, distmat_full, all_models, total_num_models,
-                sampleA_all_models, sampleB_all_models, args.sysname)
+            cutoffs_list, distmat_full, all_models, total_num_models,
+            sampleA_all_models, sampleB_all_models, args.sysname,
+            args.cores)
```

Comment:
Looks like you are extending the `cores` command line argument here, so you should change the help text for it too. Currently it says it's only affects the RMSD matrix calculations with cpu_omp; that's no longer the case.